src/EphIt/EphIt.Server/Startup.cs

```diff
@@ -46,6 +44,10 @@ public void ConfigureServices(IServiceCollection services)
                     services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
                         .AddMicrosoftIdentityWebApp(Configuration.GetSection("AzureAdApp"));
                 }
+                if(configSection["UseKestrel"] == "true")
```

Comment:
Whatever AddAuthentication is run last is the only one that is available.  I think we can use JWTBearer tokens if we ever want to use multiple auth schemes at once.